src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -285,4 +287,26 @@ public static Dimension getBrowserSize() {
   public static String getDisableCommunityPageSalesPitchDialog() {
     return getProp("disableCommunityPageSalesPitchDialog");
   }
+
+  public static HttpHost getBorderProxy() {
+
+    EnvType envType = getEnvType(getEnv());
+    File configFile = new File(getCredentialsFilePath());
+
+    switch (envType) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1301)